resources/backup_scripts/db-backup.py

```diff
@@ -127,7 +128,25 @@ def upload_new_dumps(days):
 
     print "Uploaded %d dumps to S3" % count 
 
+def create_db_dump():
+    filename = settings.backup_dir + '/' + date.today().strftime('db_backup-%d-%m-%y.sql.gz')
+    # Do not create a db dump if a file already exists and the force-dump option
+    # is false.
+    if (os.path.isfile(filename) and not settings.force_dump):
+        print "%s already exists. Aborting DB dump." % filename
+        return
+
+    drushcmd_sql_dump =  'drush -r ' + settings.docroot_dir + ' sql-dump | gzip > ' + filename
```

Comment:
@yonas could we please amend this to `drush -r ' + settings.docroot_dir + ' sql-dump --result-file=' + filename + ' --structure-tables-key=common --gzip` that should cut down on the dump file size.